chatto/message.py

```diff
@@ -37,22 +38,48 @@
 from chatto.channel import Channel
 from chatto.stream import Stream
 
+class MessageTypes(enum.Enum):
+    ChatEndedEvent = "chatEndedEvent"
+    MessageDeletedEvent = "messageDeletedEvent" 
+    NewSponsorEvent = "newSponsorEvent"
+    SponsorOnlyModeEndedEvent = "sponsorOnlyModeEndedEvent"
+    SponsorOnlyModeStartedEvent = "sponsorOnlyModeStartedEvent"
+    MemberMilestoneChatEvent = "memberMilestoneChatEvent"
+    SuperChatEvent = "superChatEvent"
+    SuperStickerEvent = "superStickerEvent"
+    TextMessageEvent = "textMessageEvent"
+    Tombstone = "tombstone"
+    UserBannedEvent = "userBannedEvent"
 
 @dataclass(eq=True, frozen=True)
 class Message:
     id: str
-    type: str
+    type: MessageTypes
```

Comment:
```suggestion
    type: MessageType
```
Remember to make this change elsewhere in the code too.